src/_pytest/hookspec.py

```diff
@@ -56,7 +56,10 @@ def pytest_addhooks(pluginmanager: "PytestPluginManager") -> None:
     """Called at plugin registration time to allow adding new hooks via a call to
     ``pluginmanager.add_hookspecs(module_or_class, prefix)``.
 
-    :param pytest.PytestPluginManager pluginmanager: The pytest plugin manager.
+    :param pluginmanager: The pytest plugin manager.
```

Comment:
This may be a silly question, but will the documentation still display the type of `pluginmanager` (based on the type annotation)? Personally, I've found that type information extremely helpful to know what kind of object each hook parameter is and what I can do with it.

Reply:
Yes, it is still displayed; sphinx knows how to get this information from the type annotation. You can see a preview of this PR rendering here to verify: https://pytest--9496.org.readthedocs.build/en/9496/reference/reference.html#bootstrapping-hooks

Reply:
Perfect :+1: Thanks!